controllers/admin/AdminCategoriesController.php

```diff
@@ -452,12 +452,12 @@ public function renderForm()
         $format = array();
         $thumb = $thumb_url = '';
         $formated_category= ImageType::getFormattedName('category');
```

Comment:
You're right. Just did it: https://github.com/PrestaShop/PrestaShop/pull/6046/commits/03f8f7d52d6af44ae6d6fa9f5aa753163139ca3a